notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>config.json is not in the repo, in other situations I saw that it was generated in the notebook, in a similar way you are generating the python script</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
config.json contains subscription id and resource group so I cannot check it in. User has to run configuration.ipynb to generate their own config.json